apps/req.c

```diff
@@ -490,9 +515,10 @@ int req_main(int argc, char **argv)
             oid_bio = BIO_new_file(p, "r");
             if (oid_bio == NULL) {
                 /*-
-                BIO_printf(bio_err,"problems opening %s for extra oid's\n",p);
-                ERR_print_errors(bio_err);
-                */
+                 * BIO_printf(bio_err,
```

Comment:
Hmm, why is this commented out? Could be useful diagnostics - perhaps enable with -verbose?

Reply:
I don't know why that diagnostics had been commented out,
but I like the idea of re-enabling it in case `verbose` is set; done.